src/lcf/rooms.395

```diff
@@ -310,7 +309,8 @@
 	<SETG INTIME <DSKDATE>>
 	<COND (<L? .MUDDLE 100>
 	       <AND <G? <LENGTH .XUNM> 2> <=? <SUBSTRUC .XUNM 0 3> "___"> <QUIT>>
-	       <SET FN <ITS-GET-NAME .XUNM>>)
+"	       <SET FN <ITS-GET-NAME .XUNM>>)"
+	       <SET FN .XUNM>)
```

Comment:
And this?

Reply:
The second hack replaces the use of GET-ITS-NAME with the use of simply the XUNAME. I did this so long ago that I’ve forgotten why I had to, but the original code crashed Muddle and this Zork. Again, I’ll look at this again and see about getting GET-ITS-NAME to work.

Reply:
But maybe we can get this merged first and open tickets for these two issues?

Reply:
Thanks for the info, I will add that.  Yes, let's keep it as is for now.  I'm running a test build to ensure things come out right.